mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q1/chr8/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q1/chr8/Plac_CON02_Q1_chr8_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/processed_bams/rna/VW-ASUPlace-Cont02_180_013_S184_L001_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chr8.gatk.called.vqsr.sv.biallelic.snp.Plac_CON02.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q1/chrX/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q1/chrX/Plac_CON02_Q1_chrX_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/processed_bams/rna/VW-ASUPlace-Cont02_180_013_S184_L001_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chrX.gatk.called.vqsr.sv.biallelic.snp.Plac_CON02.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q2/chr8/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q2/chr8/Plac_CON02_Q2_chr8_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_3samples/processed_bams_HG38/rna/Cont02_quadrant2_Sample_2_084_109_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chr8.gatk.called.vqsr.sv.biallelic.snp.Plac_CON02.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q2/chrX/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q2/chrX/Plac_CON02_Q2_chrX_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_3samples/processed_bams_HG38/rna/Cont02_quadrant2_Sample_2_084_109_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chrX.gatk.called.vqsr.sv.biallelic.snp.Plac_CON02.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q3/chr8/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q3/chr8/Plac_CON02_Q3_chr8_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_3samples/processed_bams_HG38/rna/Cont02_quadrant3_Sample_2_034_159_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chr8.gatk.called.vqsr.sv.biallelic.snp.Plac_CON02.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q3/chrX/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q3/chrX/Plac_CON02_Q3_chrX_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_3samples/processed_bams_HG38/rna/Cont02_quadrant3_Sample_2_034_159_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chrX.gatk.called.vqsr.sv.biallelic.snp.Plac_CON02.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q4/chr8/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q4/chr8/Plac_CON02_Q4_chr8_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_3samples/processed_bams_HG38/rna/Cont02_quadrant4_Sample_22_079_114_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chr8.gatk.called.vqsr.sv.biallelic.snp.Plac_CON02.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q4/chrX/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q4/chrX/Plac_CON02_Q4_chrX_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_3samples/processed_bams_HG38/rna/Cont02_quadrant4_Sample_22_079_114_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chrX.gatk.called.vqsr.sv.biallelic.snp.Plac_CON02.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q1/chr8/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q1/chr8/Plac_CON03_Q1_chr8_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/processed_bams/rna/VW-ASUPlace-Cont03_168_025_S185_L001_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chr8.gatk.called.vqsr.sv.biallelic.snp.Plac_CON03.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q1/chrX/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q1/chrX/Plac_CON03_Q1_chrX_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/processed_bams/rna/VW-ASUPlace-Cont03_168_025_S185_L001_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chrX.gatk.called.vqsr.sv.biallelic.snp.Plac_CON03.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q2/chr8/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q2/chr8/Plac_CON03_Q2_chr8_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_3samples/processed_bams_HG38/rna/Cont03_quadrant2_Sample_3_072_121_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chr8.gatk.called.vqsr.sv.biallelic.snp.Plac_CON03.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q2/chrX/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q2/chrX/Plac_CON03_Q2_chrX_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_3samples/processed_bams_HG38/rna/Cont03_quadrant2_Sample_3_072_121_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chrX.gatk.called.vqsr.sv.biallelic.snp.Plac_CON03.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q3/chr8/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q3/chr8/Plac_CON03_Q3_chr8_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_3samples/processed_bams_HG38/rna/Cont03_quadrant3_Sample_3_022_171_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chr8.gatk.called.vqsr.sv.biallelic.snp.Plac_CON03.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q3/chrX/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q3/chrX/Plac_CON03_Q3_chrX_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_3samples/processed_bams_HG38/rna/Cont03_quadrant3_Sample_3_022_171_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chrX.gatk.called.vqsr.sv.biallelic.snp.Plac_CON03.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q4/chr8/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q4/chr8/Plac_CON03_Q4_chr8_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_3samples/processed_bams_HG38/rna/Cont03_quadrant4_Sample_23_067_126_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chr8.gatk.called.vqsr.sv.biallelic.snp.Plac_CON03.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q4/chrX/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q4/chrX/Plac_CON03_Q4_chrX_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_3samples/processed_bams_HG38/rna/Cont03_quadrant4_Sample_23_067_126_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chrX.gatk.called.vqsr.sv.biallelic.snp.Plac_CON03.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q1/chr8/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q1/chr8/Plac_CON05_Q1_chr8_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/processed_bams/rna/VW-ASUPlace-Cont05-Q1RNA1_144_049_S187_L001_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chr8.gatk.called.vqsr.sv.biallelic.snp.Plac_CON06.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q1/chrX/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q1/chrX/Plac_CON05_Q1_chrX_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/processed_bams/rna/VW-ASUPlace-Cont05-Q1RNA1_144_049_S187_L001_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chrX.gatk.called.vqsr.sv.biallelic.snp.Plac_CON06.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q2/chr8/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q2/chr8/Plac_CON05_Q2_chr8_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_3samples/processed_bams_HG38/rna/Cont05_quadrant2_Sample_5_048_145_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chr8.gatk.called.vqsr.sv.biallelic.snp.Plac_CON06.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q2/chrX/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q2/chrX/Plac_CON05_Q2_chrX_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_3samples/processed_bams_HG38/rna/Cont05_quadrant2_Sample_5_048_145_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chrX.gatk.called.vqsr.sv.biallelic.snp.Plac_CON06.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q3/chr8/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q3/chr8/Plac_CON05_Q3_chr8_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_3samples/processed_bams_HG38/rna/Cont05_quadrant3_Sample_5_093_100_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chr8.gatk.called.vqsr.sv.biallelic.snp.Plac_CON06.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q3/chrX/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q3/chrX/Plac_CON05_Q3_chrX_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_3samples/processed_bams_HG38/rna/Cont05_quadrant3_Sample_5_093_100_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chrX.gatk.called.vqsr.sv.biallelic.snp.Plac_CON06.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q4/chr8/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q4/chr8/Plac_CON05_Q4_chr8_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_3samples/processed_bams_HG38/rna/Cont05_quadrant4_Sample_25_043_150_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chr8.gatk.called.vqsr.sv.biallelic.snp.Plac_CON06.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q4/chrX/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q4/chrX/Plac_CON05_Q4_chrX_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_3samples/processed_bams_HG38/rna/Cont05_quadrant4_Sample_25_043_150_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chrX.gatk.called.vqsr.sv.biallelic.snp.Plac_CON06.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q1/chr8/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q1/chr8/Plac_CON06_Q1_chr8_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/processed_bams/rna/VW-ASUPlace-Cont06-Q1RNA1_132_061_S188_L001_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chr8.gatk.called.vqsr.sv.biallelic.snp.Plac_CON05.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q1/chrX/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q1/chrX/Plac_CON06_Q1_chrX_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/processed_bams/rna/VW-ASUPlace-Cont06-Q1RNA1_132_061_S188_L001_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chrX.gatk.called.vqsr.sv.biallelic.snp.Plac_CON05.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q2/chr8/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q2/chr8/Plac_CON06_Q2_chr8_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_3samples/processed_bams_HG38/rna/Cont06_quadrant2_Sample_6_036_157_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chr8.gatk.called.vqsr.sv.biallelic.snp.Plac_CON05.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q2/chrX/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q2/chrX/Plac_CON06_Q2_chrX_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_3samples/processed_bams_HG38/rna/Cont06_quadrant2_Sample_6_036_157_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chrX.gatk.called.vqsr.sv.biallelic.snp.Plac_CON05.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q3/chr8/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q3/chr8/Plac_CON06_Q3_chr8_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_3samples/processed_bams_HG38/rna/Cont06_quadrant3_Sample_6_081_112_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chr8.gatk.called.vqsr.sv.biallelic.snp.Plac_CON05.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q3/chrX/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q3/chrX/Plac_CON06_Q3_chrX_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_3samples/processed_bams_HG38/rna/Cont06_quadrant3_Sample_6_081_112_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chrX.gatk.called.vqsr.sv.biallelic.snp.Plac_CON05.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q4/chr8/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q4/chr8/Plac_CON06_Q4_chr8_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_3samples/processed_bams_HG38/rna/Cont06_quadrant4_Sample_26_031_162_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chr8.gatk.called.vqsr.sv.biallelic.snp.Plac_CON05.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q4/chrX/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q4/chrX/Plac_CON06_Q4_chrX_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_3samples/processed_bams_HG38/rna/Cont06_quadrant4_Sample_26_031_162_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chrX.gatk.called.vqsr.sv.biallelic.snp.Plac_CON05.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q1/chr8/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q1/chr8/Plac_CON10_Q1_chr8_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/processed_bams/rna/VW-ASUPlace-Cont10-Q1RNA1_179_014_S192_L001_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chr8.gatk.called.vqsr.sv.biallelic.snp.Plac_CON10.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q1/chrX/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q1/chrX/Plac_CON10_Q1_chrX_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/processed_bams/rna/VW-ASUPlace-Cont10-Q1RNA1_179_014_S192_L001_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chrX.gatk.called.vqsr.sv.biallelic.snp.Plac_CON10.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q2/chr8/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q2/chr8/Plac_CON10_Q2_chr8_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_3samples/processed_bams_HG38/rna/Cont10_quadrant2_Sample_10_083_110_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chr8.gatk.called.vqsr.sv.biallelic.snp.Plac_CON10.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q2/chrX/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q2/chrX/Plac_CON10_Q2_chrX_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_3samples/processed_bams_HG38/rna/Cont10_quadrant2_Sample_10_083_110_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chrX.gatk.called.vqsr.sv.biallelic.snp.Plac_CON10.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q3/chr8/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q3/chr8/Plac_CON10_Q3_chr8_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_3samples/processed_bams_HG38/rna/Cont10_quadrant3_Sample_10_033_160_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chr8.gatk.called.vqsr.sv.biallelic.snp.Plac_CON10.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q3/chrX/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q3/chrX/Plac_CON10_Q3_chrX_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_3samples/processed_bams_HG38/rna/Cont10_quadrant3_Sample_10_033_160_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chrX.gatk.called.vqsr.sv.biallelic.snp.Plac_CON10.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q4/chr8/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q4/chr8/Plac_CON10_Q4_chr8_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_3samples/processed_bams_HG38/rna/Cont10_quadrant4_Sample_30_078_115_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chr8.gatk.called.vqsr.sv.biallelic.snp.Plac_CON10.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q4/chrX/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q4/chrX/Plac_CON10_Q4_chrX_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_3samples/processed_bams_HG38/rna/Cont10_quadrant4_Sample_30_078_115_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chrX.gatk.called.vqsr.sv.biallelic.snp.Plac_CON10.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q1/chr8/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q1/chr8/Plac_HDP01_Q1_chr8_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/processed_bams/rna/VW-ASUPlace-HDP01_167_026_S193_L001_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chr8.gatk.called.vqsr.sv.biallelic.snp.Plac_HDP02.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q1/chrX/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q1/chrX/Plac_HDP01_Q1_chrX_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/processed_bams/rna/VW-ASUPlace-HDP01_167_026_S193_L001_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chrX.gatk.called.vqsr.sv.biallelic.snp.Plac_HDP02.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q2/chr8/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q2/chr8/Plac_HDP01_Q2_chr8_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_3samples/processed_bams_HG38/rna/HDP01_quadrant2_Sample_11_071_122_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chr8.gatk.called.vqsr.sv.biallelic.snp.Plac_HDP02.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q2/chrX/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q2/chrX/Plac_HDP01_Q2_chrX_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_3samples/processed_bams_HG38/rna/HDP01_quadrant2_Sample_11_071_122_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chrX.gatk.called.vqsr.sv.biallelic.snp.Plac_HDP02.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q3/chr8/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q3/chr8/Plac_HDP01_Q3_chr8_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_3samples/processed_bams_HG38/rna/HDP01_quadrant3_Sample_11_021_172_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chr8.gatk.called.vqsr.sv.biallelic.snp.Plac_HDP02.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q3/chrX/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q3/chrX/Plac_HDP01_Q3_chrX_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_3samples/processed_bams_HG38/rna/HDP01_quadrant3_Sample_11_021_172_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chrX.gatk.called.vqsr.sv.biallelic.snp.Plac_HDP02.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q4/chr8/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q4/chr8/Plac_HDP01_Q4_chr8_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_3samples/processed_bams_HG38/rna/HDP01_quadrant4_Sample_31_066_127_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chr8.gatk.called.vqsr.sv.biallelic.snp.Plac_HDP02.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q4/chrX/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q4/chrX/Plac_HDP01_Q4_chrX_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_3samples/processed_bams_HG38/rna/HDP01_quadrant4_Sample_31_066_127_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chrX.gatk.called.vqsr.sv.biallelic.snp.Plac_HDP02.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q1/chr8/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q1/chr8/Plac_HDP08_Q1_chr8_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/processed_bams/rna/VW-ASUPlace-HDP08-Q1RNA1_178_015_S200_L001_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chr8.gatk.called.vqsr.sv.biallelic.snp.Plac_HDP08.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q1/chrX/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q1/chrX/Plac_HDP08_Q1_chrX_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/processed_bams/rna/VW-ASUPlace-HDP08-Q1RNA1_178_015_S200_L001_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chrX.gatk.called.vqsr.sv.biallelic.snp.Plac_HDP08.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q2/chr8/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q2/chr8/Plac_HDP08_Q2_chr8_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_3samples/processed_bams_HG38/rna/HDP08_quadrant2_Sample_18_082_111_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chr8.gatk.called.vqsr.sv.biallelic.snp.Plac_HDP08.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q2/chrX/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q2/chrX/Plac_HDP08_Q2_chrX_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_3samples/processed_bams_HG38/rna/HDP08_quadrant2_Sample_18_082_111_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chrX.gatk.called.vqsr.sv.biallelic.snp.Plac_HDP08.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q3/chr8/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q3/chr8/Plac_HDP08_Q3_chr8_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_3samples/processed_bams_HG38/rna/HDP08_quadrant3_Sample_18_032_161_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chr8.gatk.called.vqsr.sv.biallelic.snp.Plac_HDP08.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q3/chrX/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q3/chrX/Plac_HDP08_Q3_chrX_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_3samples/processed_bams_HG38/rna/HDP08_quadrant3_Sample_18_032_161_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chrX.gatk.called.vqsr.sv.biallelic.snp.Plac_HDP08.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q4/chr8/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q4/chr8/Plac_HDP08_Q4_chr8_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_3samples/processed_bams_HG38/rna/HDP08_quadrant4_Sample_38_077_116_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chr8.gatk.called.vqsr.sv.biallelic.snp.Plac_HDP08.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q4/chrX/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q4/chrX/Plac_HDP08_Q4_chrX_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_3samples/processed_bams_HG38/rna/HDP08_quadrant4_Sample_38_077_116_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chrX.gatk.called.vqsr.sv.biallelic.snp.Plac_HDP08.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q1/chr8/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q1/chr8/Plac_HDP09_Q1_chr8_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/processed_bams/rna/VW-ASUPlace-HDP09-Q1RNA1_166_027_S201_L001_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chr8.gatk.called.vqsr.sv.biallelic.snp.Plac_HDP09.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q1/chrX/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q1/chrX/Plac_HDP09_Q1_chrX_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/processed_bams/rna/VW-ASUPlace-HDP09-Q1RNA1_166_027_S201_L001_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chrX.gatk.called.vqsr.sv.biallelic.snp.Plac_HDP09.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q2/chr8/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q2/chr8/Plac_HDP09_Q2_chr8_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_3samples/processed_bams_HG38/rna/HDP09_quadrant2_Sample_19_070_123_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chr8.gatk.called.vqsr.sv.biallelic.snp.Plac_HDP09.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q2/chrX/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q2/chrX/Plac_HDP09_Q2_chrX_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_3samples/processed_bams_HG38/rna/HDP09_quadrant2_Sample_19_070_123_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chrX.gatk.called.vqsr.sv.biallelic.snp.Plac_HDP09.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q3/chr8/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q3/chr8/Plac_HDP09_Q3_chr8_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_3samples/processed_bams_HG38/rna/HDP09_quadrant3_Sample_19_020_173_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chr8.gatk.called.vqsr.sv.biallelic.snp.Plac_HDP09.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q3/chrX/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q3/chrX/Plac_HDP09_Q3_chrX_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_3samples/processed_bams_HG38/rna/HDP09_quadrant3_Sample_19_020_173_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chrX.gatk.called.vqsr.sv.biallelic.snp.Plac_HDP09.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q4/chr8/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q4/chr8/Plac_HDP09_Q4_chr8_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_3samples/processed_bams_HG38/rna/HDP09_quadrant4_Sample_39_065_128_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chr8.gatk.called.vqsr.sv.biallelic.snp.Plac_HDP09.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q4/chrX/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q4/chrX/Plac_HDP09_Q4_chrX_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_3samples/processed_bams_HG38/rna/HDP09_quadrant4_Sample_39_065_128_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chrX.gatk.called.vqsr.sv.biallelic.snp.Plac_HDP09.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q1/chr8/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q1/chr8/Plac_HDP10_Q1_chr8_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/processed_bams/rna/VW-ASUPlace-HDP10-Q1RNA1_154_039_S202_L001_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chr8.gatk.called.vqsr.sv.biallelic.snp.Plac_HDP10.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q1/chrX/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q1/chrX/Plac_HDP10_Q1_chrX_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/processed_bams/rna/VW-ASUPlace-HDP10-Q1RNA1_154_039_S202_L001_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chrX.gatk.called.vqsr.sv.biallelic.snp.Plac_HDP10.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q2/chr8/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q2/chr8/Plac_HDP10_Q2_chr8_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_3samples/processed_bams_HG38/rna/HDP10_quadrant2_Sample_20_058_135_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chr8.gatk.called.vqsr.sv.biallelic.snp.Plac_HDP10.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q2/chrX/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q2/chrX/Plac_HDP10_Q2_chrX_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_3samples/processed_bams_HG38/rna/HDP10_quadrant2_Sample_20_058_135_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chrX.gatk.called.vqsr.sv.biallelic.snp.Plac_HDP10.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q3/chr8/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q3/chr8/Plac_HDP10_Q3_chr8_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_3samples/processed_bams_HG38/rna/HDP10_quadrant3_Sample_20_008_185_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chr8.gatk.called.vqsr.sv.biallelic.snp.Plac_HDP10.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q3/chrX/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q3/chrX/Plac_HDP10_Q3_chrX_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_3samples/processed_bams_HG38/rna/HDP10_quadrant3_Sample_20_008_185_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chrX.gatk.called.vqsr.sv.biallelic.snp.Plac_HDP10.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q4/chr8/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q4/chr8/Plac_HDP10_Q4_chr8_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_3samples/processed_bams_HG38/rna/HDP10_quadrant4_Sample_40_053_140_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chr8.gatk.called.vqsr.sv.biallelic.snp.Plac_HDP10.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
mkdir -p /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q4/chrX/
sbatch --time=24:00:00 --mem=20G -n 2 --wrap="module load GATK/4.6.0.0; gatk ASEReadCounter -R /data/Wilson_Lab/references/GENCODE/GRCh38.p12.genome.XXonly/GRCh38.p12.genome.XXonly.fa --output /data/Wilson_Lab/projects/placentas_VW_ASE/02_run_asereadcounter/run_asereadcounter_correct_swapping/HISAT/Q4/chrX/Plac_HDP10_Q4_chrX_females.tsv --input /data/Wilson_Lab/projects/placenta_multiomics/valleywise_3samples/processed_bams_HG38/rna/HDP10_quadrant4_Sample_40_053_140_HISAT_pair_trim_sort_mkdup_rdgrp_XX.bam --variant /data/Wilson_Lab/projects/placenta_multiomics/valleywise_pilot/vqsr/chrX.gatk.called.vqsr.sv.biallelic.snp.Plac_HDP10.het.vcf --min-depth-of-non-filtered-base 1 --min-mapping-quality 10 --min-base-quality 10"
